server/base/src/main/java/org/apache/accumulo/server/manager/state/TabletManagementParameters.java

```diff
@@ -125,14 +126,17 @@ private TabletManagementParameters(JsonData jdata) {
       return Map.copyOf(resourceGroups);
     });
     this.canSuspendTablets = jdata.canSuspendTablets;
-    this.volumeReplacements = jdata.volumeReplacements.stream()
-        .collect(toUnmodifiableMap(Pair::getFirst, Pair::getSecond));
+    this.volumeReplacements = jdata.volumeReplacements;
```

Comment:
This map seems to come from GSON, not sure if its modifiable.  If its could use Map.copyof or the following to make it unmodifiable. 

```suggestion
    this.volumeReplacements = Collections.unmodifiableMap(jdata.volumeReplacements);
```